modules/shortcodes/flickr.php

```diff
@@ -36,22 +42,27 @@ function flickr_embed_to_shortcode( $content ) {
 		htmlspecialchars( $regexp, ENT_NOQUOTES )
 	);
 
-	foreach ( array( 'regexp', 'regexp_ent' ) as $reg ) {
-		if ( ! preg_match_all( $$reg, $content, $matches, PREG_SET_ORDER ) ) {
+	foreach ( compact( 'regexp', 'regexp_ent' ) as $reg => $regexp ) {
+		if ( ! preg_match_all( $regexp, $content, $matches, PREG_SET_ORDER ) ) {
 			continue;
 		}
 		foreach ( $matches as $match ) {
 			$params = $match[2] . $match[3];
 
-			if ( 'regexp_ent' == $reg ) {
+			if ( 'regexp_ent' === $reg ) {
 				$params = html_entity_decode( $params );
 			}
 
 			$params = wp_kses_hair( $params, array( 'http' ) );
-			if ( ! isset( $params['type'] ) || 'application/x-shockwave-flash' != $params['type']['value'] || ! isset( $params['flashvars'] ) ) {
+			if (
+				! isset( $params['type'] )
+				|| 'application/x-shockwave-flash' !== $params['type']['value']
+				|| ! isset( $params['flashvars'] )
+			) {
 				continue;
 			}
 
+			$flashvars = array();
```

Comment:
Not really needed, but we have to declare the variable as phpcs doesn't get that it's actually done in `wp_parse_str` below.